### ISSUE TITLE
Data::Util dependency

### ISSUE THREAD

**Issue by vividsnow** [gnustavo/JIRA-REST#4]
Hi

Looks like Data::Util dont work under perl-5.22.
Maybe replace "Data::Util::is_instance" with https://metacpan.org/pod/UNIVERSAL#isa?

**Comment by gnustavo** [comment]
Hi. Thanks for the tip.

Besides is_instance I use is_string and is_hash_ref. I looked around for other argument validation modules and became frustrated with the ones I found: Params::Util, Smart::Args, Data::Validator, Params::Check, Validate::TIny, Type::Params, and Params::Validate. They are either too heavy (depending on Moose ou Mouse or Moo), too simple, or somewhat arkane to use. ;-)

Since I only use three, I'm thinking about doing it directly and avoiding yet another dependency.

I'll sleep over it to think more.
